test/test_environment.py

```diff
@@ -13,18 +14,21 @@
 from asv import util
 
 
+WIN = (os.name == "nt")
+
+
 try:
     util.which('python2.7')
     HAS_PYTHON_27 = True
-except RuntimeError:
-    HAS_PYTHON_27 = False
+except (RuntimeError, IOError):
+    HAS_PYTHON_27 = (sys.version_info[:2] == (2, 7))
```

Comment:
This logic doesn't seem right.  `HAS_PYTHON_27` is supposed to indicate that python2.7 is available somewhere on the `PATH`, not to indicate what the present version of Python is.